artifactory/commands/dotnet/dotnetcommand.go

```diff
@@ -18,19 +18,20 @@ import (
 	"strings"
 )
 
-const SourceName = "JFrogCli"
+const (
+	SourceName        = "JFrogCli"
```

Comment:
It looks like this const can be made private.

Reply:
It is used by the tests in the CLI

Reply:
Got it. NP.